requirements/local-docs-build.txt

```diff
@@ -0,0 +1,23 @@
+# Base packages 
+docutils==0.14
+Sphinx==1.8.3
+sphinx_rtd_theme==0.4.2
+sphinx-tabs==1.1.10
+# Required to avoid Transifex error with reserved slug
+# https://github.com/sphinx-doc/sphinx-intl/pull/27
+git+https://github.com/agjohnson/sphinx-intl.git@7b5c66bdb30f872b3b1286e371f569c8dcb66de5#egg=sphinx-intl
+
+Pygments==2.3.1
+
+mkdocs==1.0.4
+Markdown==3.0.1
+
+# Docs
+sphinxcontrib-httpdomain==1.7.0
+sphinx-prompt==1.0.0
```

Comment:
Wasn't aware of that these are only for RTD docs.

> these should probably go into a different reqs file maybe.

Since these are RTD-only requirements, we can put it back in the `pip.txt`

Reply:
Well, they are not required to "run Read the Read software" but to build the docs of it.

So, basically, we should use this `requirements/docs.txt` file in our own projects under readthedocs.io (https://readthedocs.org/projects/docs/)

Reply:
Okay.
I have separated them in `docs.txt` and also add `-r local-docs-build.txt` to install the other dependencies.